### PR TITLE
define type DfStat,	remove dfColumnSpecs and refactor

### PR DESCRIPTION
**Pull request by Songmu**
follow on #262

### CODE REVIEW

**Comment by astj** [comment]
[memo]
Changes after #262: https://github.com/mackerelio/mackerel-agent/compare/1d3d98765ed9f4557d9e643ca4e5e3b227a90376...df

File: metrics/filesystem.go [hunk 1]
```diff
@@ -14,38 +14,28 @@ type FilesystemGenerator struct {
...
 
-var dfColumnSpecs = []util.DfColumnSpec{
-	{Name: "size", IsInt: true},
-	{Name: "used", IsInt: true},
-}
-
 var sanitizerReg = regexp.MustCompile(`[^A-Za-z0-9_-]`)
 
 // Generate the metrics of filesystems
 func (g *FilesystemGenerator) Generate() (Values, error) {
```
Comment (astj) [inline metrics/filesystem.go]: I found that tests for `Generate()` is so simple 😱
https://github.com/mackerelio/mackerel-agent/blob/master/metrics/filesystem_test.go

File: util/filesystem.go [hunk 2]
```diff
@@ -18,6 +19,15 @@ import (
 	"github.com/mackerelio/mackerel-agent/logging"
 )
 
+type DfStat struct {
+	Name      string
+	Blocks    uint64
```
Comment (astj) [inline util/filesystem.go]: [suggestion]
"Blocks" and "Capacity" are named "size" and "percent_used" in spec. Those mismatch are confusing, so how about using same names as spec?

**Comment by astj** [comment]
Reviewed.

**Comment by Songmu** [comment]
The field names of `dfStat` are same with the result of `df -P`.

**Comment by astj** [comment]
https://github.com/mackerelio/mackerel-agent/pull/263#issuecomment-239995262

Okay, I understand 🙆

**Pull request merged**